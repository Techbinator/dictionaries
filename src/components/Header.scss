.header {
  flex-direction: row;
  display: flex;
  justify-content: space-evenly;
  background: white;
  padding: 20px;
  box-shadow: 0 4px 8px #00000030;
  .menu {
    display: flex;
    justify-content: flex-end;
    flex-grow: 1;
    a {
      display: block;
      color: black;
      padding: 10px;
      border: 1px solid #ccc;
      box-shadow: 0 4px 8px #00000030;
      &.active,
      &:hover {
        background-color: #ccc;
      }
    }
  }
  img {
    width: auto;
    max-height: 33px;
  }
}
