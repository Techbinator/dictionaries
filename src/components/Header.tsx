import React from "react";
import { NavLink } from "react-router-dom";

import logo from "../assets/images/logo.png";
import "./Header.scss";

export default () => (
  <header className="header">
    <img src={logo} />
    <div className="menu">
      <NavLink exact to="/">
        Dataset
      </NavLink>
      <NavLink exact to="/dictionaries">
        Dictionaries
      </NavLink>
    </div>
  </header>
);
