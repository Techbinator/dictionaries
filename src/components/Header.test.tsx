import React from "react";
import Header from "./Header";
import { shallow } from "enzyme";

describe("Header component", () => {
  it("should render snapshot with default values", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
