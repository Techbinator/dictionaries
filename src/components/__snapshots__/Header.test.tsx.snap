// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render snapshot with default values 1`] = `
<header
  className="header"
>
  <img
    src="logo.png"
  />
  <div
    className="menu"
  >
    <NavLink
      activeClassName="active"
      aria-current="page"
      exact={true}
      to="/"
    >
      Dataset
    </NavLink>
    <NavLink
      activeClassName="active"
      aria-current="page"
      exact={true}
      to="/dictionaries"
    >
      Dictionaries
    </NavLink>
  </div>
</header>
`;
